src/test/resources/compliantJsonArray.json

```diff
@@ -0,0 +1,317 @@
+[
```

Comment:
Please remove all potential PII from this file. Name, address, phone, and email values should be replaced with e.g. "Abc Def", "123 Main St", "123-456-7890", "abc@def.com", etc.

Reply:
Done.